src/components/atoms/AtomDatePicker.vue

```diff
@@ -25,13 +27,22 @@ export default {
     emits: ['changed'],
     data() {
         return {
-            date: new Date(),
+            date:
+                this.assignedDate != ''
+                    ? new Date(this.assignedDate)
+                    : new Date(),
         };
     },
     methods: {
         onChange(val) {
             this.$emit('changed', val);
         },
+        istFormatter(date) {
```

Comment:
This variable name suggests a boolean return value, but it actually returns a date. Could we rename it to something more meaningful?

Reply:
its 'ist'Formatter  (Indian Standard Time)